frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -37,14 +37,8 @@ const LogsTabStatus: React.FC<LogsTabStatusProps> = ({ error, loaded, refresh, o
         The log refreshes every {Math.floor(LOG_REFRESH_RATE / 1000)} seconds and displays the
         latest {LOG_TAIL_LINES} lines. Exceptionally long lines are abridged. To view the full log
         for this task, you can{' '}
-        <Button
-          isDisabled={!!onDownload}
-          variant="link"
-          isInline
-          component="span"
-          onClick={onDownload}
-        >
-          download the selected container logs
+        <Button variant="link" isInline component="span" onClick={onDownload}>
+          download all step logs
```

Comment:
Fixed this according to recent mocks

Reply:
You removed the disable selection... `onDownload` is an optional prop. Why make a link that does nothing when you click on it? We have to respect the types here... so either you over compensated or you failed to update the types on the component.

TypeScript is the one true answer to if we should do something, it should reflect the absolute state we expect.